Makefile

```diff
@@ -153,3 +154,14 @@ usage:
 	@echo "    clean     - cleans the dist dir"
 	@echo "    distclean - cleans the dist dir and the dependencies"
 
+
+# Rules to attach checksums to JavaScript source in the header
+app.html: app.html.in .make-script-block
+	sed -e '/%SCRIPTS%/ {' -e 'r .make-script-block' -e 'd' -e '}' -i app.html
```

Comment:
I don't agree with doing in-place edit (i.e., `-i` option) for this use.

We want a clear distinction between the _source file_ (`app.html.in`) and the _output file_ (`app.html`), and the source file should not be modified by the make procedure. Also the output file should be in the `dist` subdir.